frontend/src/settings/beta/SettingsProfileTab.tsx

```diff
@@ -0,0 +1,95 @@
+import ContentCopy from '@mui/icons-material/ContentCopy'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Chip from '@mui/material/Chip'
+import Divider from '@mui/material/Divider'
+import IconButton from '@mui/material/IconButton'
+import Stack from '@mui/material/Stack'
+import { useTheme } from '@mui/material/styles'
+import Tooltip from '@mui/material/Tooltip'
+import Typography from '@mui/material/Typography'
+import { useState } from 'react'
+import { User } from 'types/types'
+
+function SettingsProfileTab({ user }: { user: User }) {
+  const theme = useTheme()
+  const [displayToken, setDisplayToken] = useState(false)
```

Comment:
These state vars are confusingly similar in name imo. How about `token` and `showToken`?